models/qrCodeAuth.js

```diff
@@ -0,0 +1,41 @@
+const firestore = require("../utils/firestore");
+const QrCodeAuthModel = firestore.collection("QrCodeAuth");
+const userModel = firestore.collection("users");
+const USER_DOES_NOT_EXIST_ERROR = "User does not exist.";
+/**
+ * Stores the user device info
+ *
+ * @param userDeviceInfoData { Object }: User device info data object to be stored in DB
+ * @return {Promise<{userDeviceInfoData|Object}>}
+ */
+const storeUserDeviceInfo = async (userDeviceInfoData) => {
+  try {
+    const { user_id: userId } = userDeviceInfoData;
+    const user = await userModel.doc(userId).get();
+    if (user.data()) {
+      await QrCodeAuthModel.doc(userId).set(userDeviceInfoData);
```

Comment:
are we keeping the document id in this collection = userId ?

Reply:
in this collection `user_id` is the same as `id` in the users collection

Reply:
yes its the same id